docker/test/integration/runner/compose/docker_compose_ldap.yml

```diff
@@ -14,3 +14,8 @@ services:
             LDAP_PORT_NUMBER: ${LDAP_INTERNAL_PORT:-1389}
         ports:
             - ${LDAP_EXTERNAL_PORT:-1389}:${LDAP_INTERNAL_PORT:-1389}
+        healthcheck:
+            test: "ldapsearch -x -b dc=example,dc=org cn > /dev/null"
```

Comment:
Did you take it from somewhere?

Reply:
Yes, from a random page in the internet https://gist.github.com/Wildcarde/600c4848370efdd8a767d924686f4a52

Reply:
But I replaced `dc=nichebuilt,dc=club` with `dc=example,dc=org`